recipe/meta.yaml

```diff
@@ -33,15 +36,15 @@ requirements:
     - nlohmann_json =3.1.2
     - numpy
     - pip
-    - python
+    - python =3.9
```

Comment:
Should probably revert this though.

Reply:
Yes, there are some changes in here to limit the CI just to save some trees. I don't have a mac so I did all my testing in the CI here. I will take them out before merging.